Library/OcAppleDiskImageLib/OcAppleDiskImageLibInternal.c

```diff
@@ -239,9 +239,9 @@ InternalParsePlist (
     goto DONE_ERROR;
   }
 
-  Result = OcOverflowMulU32 (NumDmgBlocks, sizeof (*DmgBlocks), &DmgBlocksSize);
-  if (Result) {
-    return FALSE;
+  Result = !OcOverflowMulU32 (NumDmgBlocks, sizeof (*DmgBlocks), &DmgBlocksSize);
+  if (!Result) {
```

Comment:
Ugh, double negation? Please do not do it. This is very hard to read for no reason.

Reply:
The `Result` value is checked at DONE_ERROR, and it should be FALSE in case of error.

Reply:
Ok, but maybe make it

```
Result = Oc…
if (Result) {
  Result = FALSE; ///< Result should be false in case of error.
  goto DONE_ERROR;
}
```

I also found another issue with this code:

```
    if (!Result || (BlockDictChildDataSize < sizeof (*Block))) {
      goto DONE_ERROR;
    }
```

Reply:
But then Result will be FALSE both in case of error and no error. I think that a variable should have consistent semantics across the whole function.